### DISCUSSION TITLE
大佬 这个块能否加个插槽

### DISCUSSION THREAD

**Discussion by lovesL** [tolking/element-pro-components#457] [Ideas]
![image](https://github.com/tolking/element-pro-components/assets/27960630/20c00345-1ba6-4f5e-8459-2339e39ad06d)
我想自定义一下这块按钮 然后加个按钮

**Comment by tolking** [comment]
不是有插槽吗，自己看文档。忘记写到demo里面了

**Reply by lovesL** [reply]
..我眼瞎 我没找到大佬 复制一下

**Comment by tolking** [comment]
| search-menu-left    | 搜索表单按钮左侧                         | { loading: boolean }                                                             |
| search-menu-right   | 搜索表单按钮右侧                         | { loading: boolean }                                                             |
